src/main/flight/pid.c

```diff
@@ -766,7 +780,7 @@ void pidController(const pidProfile_t *pidProfile, const rollAndPitchTrims_t *an
         pidData[axis].P = pidData[axis].P * getThrottlePAttenuation() * setPointPAttenuation[axis];
         pidData[axis].I = temporaryIterm[axis] * getThrottleIAttenuation() * setPointIAttenuation[axis]; // you can't use pidData[axis].I to calculate iterm or with tpa you get issues
         pidData[axis].D = pidData[axis].D * getThrottleDAttenuation() * setPointDAttenuation[axis];
-        const float pidSum = pidData[axis].P + pidData[axis].I + pidData[axis].D;
+        const float pidSum = pidData[axis].P + pidData[axis].I + pidData[axis].D + yawFeedForward;
```

Comment:
I don't like the fact that pidSum isn't the P-I-D anymore, actually... Also it's a little confusing seeing a *yaw*FeedForward being added when the axis is not yaw (yes I know it would be 0 i  that case). There isn't a way to obtain the same result in a more elegant way? E.g. adding the component to some other already existing one?

Reply:
PIDF is common for a pidsum. bf does that as well. I wouldnt want to add this to p I or d as it would ruin logging them. Perhaps I can put the yaw check right at the pidsum instead though so it's a little easier to read.